### PR TITLE
Deps upgrade

### PR DESCRIPTION
**Pull request by rhyslbw**
This branch was missed from merging in the original release. With a defect discovered in #63, we need to run through QA again (with Docker included in the scope!), so now is the time to assess it.

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -47,15 +53,15 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-db-sync
-  tag: 1185967393d5052f3c5af540e1e1f058599019cf
-  --sha256: 0irjm0ji7kdmaa25d9248kcy78a19akz82zvrrxh78r496i0cch2
+  tag: fe1bcc55c857cbebbf331d81383fd6608f723fa1
```
Comment (KtorZ) [inline cabal.project:56]: Can we make sure that the same revisions are also updated in the stack's snapshot?

**Comment by rhyslbw** [comment]
@intricate Please provide some context in the PR description. The branch was found without a PR being raised, and the looks of things, is needed. We're preparing another release now, so would be good to include. Please detail any areas for QA to focus on,

**Comment by shmish111** [comment]
This PR updates dependencies to what was the latest db-sync at the time (friday evening).

File: cabal.project (hunk 1 above)
Comment (tatyanavych) [reply cabal.project:56]: WE say in release notes that it is compatible with cardano-db-sync 3.0.0 that has the later commit https://github.com/input-output-hk/cardano-db-sync/commit/beca3f8e1868bcc12da2ca4bbf0f97708fefc6c3.

File: explorer-api/src/Explorer/Web/ClientTypes.hs [hunk 2]
```diff
@@ -291,7 +291,7 @@ instance ToJSON CAddressBalanceError where
 --------------------------------------------------------------------------------
 
 toCHash :: Hash h a -> CHash
-toCHash = CHash . T.decodeLatin1 . B16.encode . getHash
+toCHash = CHash . T.decodeLatin1 . B16.encode . hashToBytes
```
Comment (intricate) [inline explorer-api/src/Explorer/Web/ClientTypes.hs:294]: If we update to the latest `cardano-base`, we can now use `hashToTextAsHex`:

```suggestion
toCHash = CHash . hashToTextAsHex
```

File: explorer-api/src/Explorer/Web/ClientTypes.hs [hunk 3]
```diff
@@ -42,7 +42,7 @@ module Explorer.Web.ClientTypes
        ) where
 
 import Cardano.Crypto.Hash.Class
-    ( Hash (getHash) )
+    ( Hash(..), hashToBytes )
```
Comment (intricate) [inline explorer-api/src/Explorer/Web/ClientTypes.hs:45]: Constructor import doesn't seem necessary: 

```suggestion
    ( Hash, hashToBytes )
```

File: explorer-api/src/Explorer/Web/Query.hs [hunk 4]
```diff
@@ -127,7 +127,7 @@ queryBlockSummary blkHash = do
 
 querySlotTimeSeconds :: MonadIO m => Word64 -> SqlPersistT m (Maybe POSIXTime)
 querySlotTimeSeconds slotNo =
-  either (const Nothing) Just <$> querySlotPosixTime slotNo
+  either (const Nothing) (Just . utcTimeToPOSIXSeconds) <$> querySlotUtcTime slotNo
```
Comment (intricate) [inline explorer-api/src/Explorer/Web/Query.hs:130]: :+1:

File: submit-api/src/Cardano/TxSubmit/Web.hs [hunk 5]
```diff
@@ -178,4 +178,4 @@ renderMediumTxId (TxId hash) = renderMediumHash hash
 -- | Render the first 16 characters of a hex-encoded hash.
 renderMediumHash :: Crypto.Hash crypto a -> Text
 renderMediumHash =
-  Text.take 16 . Text.decodeLatin1 . Crypto.getHashBytesAsHex
+  Text.take 16 . Text.decodeLatin1 . Crypto.hashToBytesAsHex
```
Comment (intricate) [inline submit-api/src/Cardano/TxSubmit/Web.hs:181]: If you update the `cardano-base` dependency to the latest:

```suggestion
  Text.take 16 . Crypto.hashToTextAsHex
```

File: cabal.project [hunk 6]
```diff
@@ -143,36 +149,36 @@ source-repository-package
...
+  tag: 7d795c3040ea7785812efa1c97864bbb41b15d3e
+  --sha256: 130i0yj4y9br1m2bhisi6wni3f40i31nfhg878hv0kwi17chl9sz
   subdir: binary
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
-  tag: c454b6e791ee2fe84508b4d5ed2c4dedafb7dce4
-  --sha256: 01m5jq6gsym3j4v85lv01n6f8480lglrb0n2mv87aqm5ksh4di9y
+  tag: 7d795c3040ea7785812efa1c97864bbb41b15d3e
```
Comment (intricate) [inline cabal.project:159]: @shmish111: If we're using `hashToTextAsHex`, we need to update this dependency.

**Pull request merged**